web_src/less/_base.less

```diff
@@ -103,13 +103,14 @@
   --color-menu: #ffffff;
   --color-card: #ffffff;
   --color-markdown-table-row: #00000008;
-  --color-markdown-code-block: #00000008;
+  --color-markdown-code-block: #00000010;
   --color-button: #ffffff;
   --color-code-bg: #ffffff;
-  --color-markdown-code-block: #00000010;
```

Comment:
FYI this is just a duplicate variable. It was fixed on master earlier with a dependency update which was not backported.